src/app/zap-templates/templates/app/cluster-structs-header.zapt

```diff
@@ -0,0 +1,22 @@
+{{> header}}
+#pragma once
+
+#include <clusters/shared/Structs.h>
```

Comment:
OK, so this is relying on clusters/shared/Structs.h pull in the headers declaring the various types that struct items can be?  But then why did events include those headers directly?

Reply:
Added actual includes - we should not rely on transitivity if at all possible (although some things realistically leak ... we do not include CHIPError in all our cpp files for example...)